tests/test_issue18_live_prereq_integration.py

```
@@ -0,0 +1,84 @@
+        provisioning_mode="byoc_lite",
+        status="ready",
+        customer_role_arn=_required("SPARKPILOT_LIVE_CUSTOMER_ROLE_ARN"),
+        region=os.getenv("SPARKPILOT_LIVE_REGION", "us-east-1"),
+        eks_cluster_arn=_required("SPARKPILOT_LIVE_EKS_CLUSTER_ARN"),
+        eks_namespace=_required("SPARKPILOT_LIVE_EKS_NAMESPACE"),
+        emr_virtual_cluster_id=os.getenv("SPARKPILOT_LIVE_EMR_VIRTUAL_CLUSTER_ID", "vc-live"),
+    )
+
+    checks: list[dict[str, object]] = []
+
+    def add_check(*, code: str, status_value: str, message: str, remediation: str | None = None, details=None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for private function `add_check`

Add return type annotation: `None`

(ANN202)